mobile-app/app/screens/AppNavigator/screens/Dex/DexScreen.tsx

```diff
@@ -171,161 +196,258 @@ interface DexItem<T> {
+          testID='pool_pair_row'
+        >
+          <View style={tailwind('flex-row items-center')}>
+            <PoolPairIcon symbolA={symbolA} symbolB={symbolB} />
+            <ThemedText
+              style={tailwind('text-base font-medium')}
+              testID={`your_symbol_${symbol}`}
+            >
+              {symbol}
+            </ThemedText>
+          </View>
+
+          <PoolPairInfoDetails type='available' pair={pair} tokenATotal={pair?.tokenA.reserve} tokenBTotal={pair?.tokenB.reserve} testID='available' />
+          <View style={tailwind('flex-row mt-4 flex-wrap')}>
+            <PoolPairActionButton
+              name='add'
+              onPress={() => onAdd(pair)}
+              pair={symbol}
+              label={translate('screens/DexScreen', 'ADD LIQUIDITY')}
+            />
+            <PoolPairActionButton
+              name='swap-horiz'
+              onPress={() => navigation.navigate({
+                name: 'PoolSwap',
+                params: { pair },
+                merge: true
+              })}
+              pair={symbol}
+              label={translate('screens/DexScreen', 'SWAP TOKENS')}
+            />
+          </View>
+        </ThemedView>
+      )
+    }}
+    />
+)
+}
 
-      <View style={tailwind('flex-row mt-4 flex-wrap -mr-2')}>
-        <PoolPairActionButton
-          name='add'
-          onPress={onAdd}
-          pair={symbol}
-          label={translate('screens/DexScreen', 'ADD MORE')}
-        />
+function PoolPairInfoDetails (props: {type: 'available' | 'your', pairAmount?: string, pair: PoolPairData | undefined, tokenATotal: string, tokenBTotal: string, testID: string}): JSX.Element {
+  const { type, pair, pairAmount, tokenATotal, tokenBTotal } = props
+  const pairSymbol = (pair?.tokenA.displaySymbol !== undefined && pair?.tokenB.displaySymbol !== undefined) ? `${pair?.tokenA?.displaySymbol}-${pair?.tokenB?.displaySymbol}` : ''
+  const decimalScale = type === 'available' ? 2 : 8
 
-        <PoolPairActionButton
-          name='remove'
-          onPress={onRemove}
-          pair={symbol}
-          label={translate('screens/DexScreen', 'REMOVE')}
-        />
-      </View>
-    </ThemedView>
-  )
+  return (
+    <View style={tailwind('mt-1 -mb-1 flex flex-row flex-wrap')}>
+      {
+        pair !== undefined && (
+          <>
+            {
+              type === 'your' && pairAmount !== undefined && (
+                <PoolPairInfoLine
+                  label={translate('screens/DexScreen', 'Pooled {{symbol}}', { symbol: pairSymbol })}
+                  value={{
+                    text: pairAmount,
+                    decimalScale: 8,
+                    testID: `your_${pairSymbol}`
+                  }}
+                />
+              )
+            }
+            <PoolPairInfoLine
+              label={translate('screens/DexScreen', `${type === 'available' ? 'Total pooled' : 'Pooled'} {{symbol}}`, { symbol: pair?.tokenA?.displaySymbol })}
```

Comment:
From figma the label has `Your ...`, e.g.: `Your pooled dBTC`. I think we should follow that to avoid confusion with `Total pooled dBTC`?
![image](https://user-images.githubusercontent.com/40191153/140493896-a1054671-99ad-43d2-91b0-07200d425a7e.png)